test/chunking-form/samples/amd-id-code-split/_config.js

```diff
@@ -0,0 +1,17 @@
+module.exports = {
+	solo: true,
+	description: 'handles amd.id with the [id] pattern',
+	options: {
+		input: ['main'],
+		output: {
+			name: 'outputName',
+			amd: {
+				id: 'something/[id]'
+			}
+		}
+	},
+	additionalFormats: ['umd'],
+	runAmd: exports => {
```

Comment:
You will find that `exports` is undefined for you. The reason is that indeed even though it loads the `main` chunk, it does not load the main module because the id does not match. If you add logs to your code, you will see that no code is executed.

To fix this, we should probably extend the runAmd object so that you can specify also the id of the main entry if it is not `main`, but it will require some more digging how to handle this in requirejs.